HomeWorks/java/lesson6/Task2HW6.java

```diff
@@ -0,0 +1,21 @@
+package lesson6;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class Task2HW6 {
+
+    public static void main(String[] args) {
+        HashMap<Integer, String> map = new HashMap<>();
+        List<String> listOfBooks = new ArrayList<>();
+        listOfBooks.add("Misery");
+        listOfBooks.add("Carrie");
+        listOfBooks.add("The Shining");
+        listOfBooks.add("The Green Mile");
+        listOfBooks.add("The Dead Zone");
+
+        listOfBooks.forEach(el -> map.put(listOfBooks.indexOf(el), el));
```

Comment:
ok but it's better to create a method for it

Reply:
public static HashMap<Integer, String> createBookMap(ArrayList bookList) {
HashMap<Integer, String> bookMap = new HashMap<>();

    for (int i = 0; i < bookList.size(); i++) {
        bookMap.put(i, bookList.get(i));
    }
    return bookMap;
}